test/integration/patients/patient/patient-flow.js

```diff
@@ -162,69 +164,67 @@ context('patient flow page', function() {
-        fx.data.attributes.updated_at = testTs();
-        fx.data.relationships.state.data.id = '33333';
-
-        const flowActions = _.sample(fx.data.relationships.actions.data, 3);
-
-        _.each(flowActions, (action, index) => {
-          action.id = `${ index + 1 }`;
+        fx.data = mergeJsonApi(testFlow, {
+          attributes: {
+            updated_at: testTs(),
+          },
+          relationships: {
+            state: getRelationship(stateInProgress),
+          },
         });
 
-        fx.data.relationships.actions.data = flowActions;
-
         return fx;
       })
       .routePatientByFlow()
       .routeFlowActions(fx => {
-        fx.data = _.first(fx.data, 3);
-
-        fx.data[0].id = '1';
-        fx.data[0].attributes.name = 'First In List';
-        fx.data[0].attributes.due_date = testDateSubtract(1);
-        fx.data[0].attributes.created_at = testTsSubtract(1);
-        fx.data[0].attributes.sequence = 1;
-        fx.data[0].attributes.outreach = 'patient';
-        fx.data[0].attributes.sharing = 'sent';
-        fx.data[0].relationships.patient.data.id = '1';
-        fx.data[0].relationships.state.data.id = '22222';
-        fx.data[0].relationships.owner.data = {
-          id: '22222',
-          type: 'teams',
-        };
-        fx.data[0].relationships.form.data = { id: '1' };
-        fx.data[0].relationships.files = { data: [{ id: '1' }] };
-
-        fx.data[1].id = '2';
-        fx.data[1].attributes.name = 'Third In List';
-        fx.data[1].attributes.due_date = testDateAdd(1);
-        fx.data[1].attributes.created_at = testTsSubtract(3);
-        fx.data[1].attributes.sequence = 3;
-        fx.data[1].relationships.patient.data.id = '1';
-        fx.data[1].relationships.state.data.id = '55555';
-        fx.data[1].relationships.owner.data = {
-          id: '33333',
-          type: 'teams',
-        };
-
-        fx.data[2].id = '3';
-        fx.data[2].attributes.name = 'Second In List';
-        fx.data[2].attributes.due_date = testDateAdd(2);
-        fx.data[2].attributes.created_at = testTsSubtract(2);
-        fx.data[2].attributes.sequence = 2;
-        fx.data[2].relationships.patient.data.id = '1';
-        fx.data[2].relationships.state.data.id = '33333';
-        fx.data[2].relationships.owner.data = {
-          id: '44444',
-          type: 'teams',
-        };
-
-        fx.included = _.reject(fx.included, { type: 'flows' });
-
-        fx.included.push({ id: '1', type: 'forms', attributes: { name: 'Test Form' } });
+        fx.data = [
+          getAction({
+            id: '1',
+            attributes: {
+              name: 'First In List',
+              due_date: testDateSubtract(1),
+              created_at: testTsSubtract(1),
+              sequence: 1,
+              outreach: 'patient',
+              sharing: 'sent',
+            },
+            relationships: {
+              flow: getRelationship(testFlowId, 'flows'),
```

Comment:
just `getRelationship(testFlow)` here and below

Reply:
Yup, updated 👍